FahadAPoem.txt

```diff
@@ -0,0 +1,4 @@
+Roses are red
+Violets are blue 
+Approve this pull request 
+the end
```

Comment:
This is great! Can you capitalize `the` on this line for consistency please?

Reply:
Fixed it